### PR TITLE
Update the banners at the top and bottom of the homepage (#7684)

### LINKED ISSUE #7684: Update the banners at the top and bottom of the homepage
## Description

Update the banners at the top and bottom of the homepage in all Teir 1 languages.

Assets will be attached to [this Trello card](https://trello.com/c/dK19hesi/538-web-work-coding-new-headers-on-mozorg-in-en-de-and-fr-3) in the NA Firefox Brand team's sprint board.

[Bug 1569237](https://bugzilla.mozilla.org/show_bug.cgi?id=1569237)

---

#### Tasks
- [x] Update all strings
- [x] Remove old assets
- [x] Add new assets
- [x] Tweak typography
- [x] String change on /accounts to support the "family" language change.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Update copy in EN, DE, FR
- Remove sticky banners description and subtitle options for (not used by new design)
- add new assets and remove old ones
- change wordmarks to images

## Issue / Bugzilla link

Fix #7684 

## Testing

- There are different banners in Firefox and in not-Firefox
- Changes made in EN, DE, and FR.

### CODE REVIEW

File: media/css/mozorg/home/home-2018.scss [hunk 1]
```diff
@@ -79,136 +77,83 @@ h4 {
...
+            background-image: url('/media/img/home/2018/browser-high-res.png'), url('/media/img/home/2018/bg-primary.svg');
+        }
     }
 
-    .fxaccount-primary-cta .mzp-l-content {
-        @include at2x('/media/img/firefox/accounts/fxa-hero-img.png', 584px, 389px);
-        background-position: 560px bottom;
-        padding-bottom: $spacing-lg;
+    .fxaccount-primary-cta {
+        @include at2x('/media/img/home/2018/home.jpg', 815px, 362px);
```
Comment (craigcook) [inline media/css/mozorg/home/home-2018.scss]: This image is 404, the file type should be .png (though might jpg be a smaller file? Does it need transparency?)

**Comment by craigcook** [comment]
Looking at it more, it feels like the image should come down further, like it should be aligned to the bottom and overflow at the top (though that means a larger image that doesn't cut off at the top).

**Comment by craigcook** [comment]
> Looking at it more, it feels like the image should come down further, like it should be aligned to the bottom and overflow at the top (though that means a larger image that doesn't cut off at the top).

image

Perfect!

File: media/css/mozorg/home/home-2018.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/mozorg/home/home-2018.scss]: I converted these to .jpg like I intended to and cropped out some white space so it appears a bit bigger now.

**Comment by stephaniehobson** [comment]
I agree an image that goes to the bottom and overflows at the top would be better.

We need to launch with this one. Hopefully it's less objectionable new that it's displaying a little bigger.

**Pull request merged**